helm/flowforge/templates/_helpers.tpl

```diff
@@ -172,3 +180,16 @@ Configure broker domain
     {{ printf "%s.%s" "mqtt" .Values.forge.domain }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Configure emqx bootstrap api secret
+*/}}
+{{- define "emqx.bootstrapApiKeySecret" -}}
+{{- $secretName := "emqx-config-secrets" }}
+{{- $existingSecret := (lookup "v1" "Secret" .Release.Namespace $secretName) | default dict }}
+{{- if and $existingSecret.data (hasKey $existingSecret.data "api_key_secret") }}
+{{- printf $existingSecret.data.api_key_secret | b64dec }}
+{{- else }}
+{{- randAlphaNum 32 -}}
+{{- end }}
+{{- end -}}
```

Comment:
Need to think about this, as we will be needing to use this secret in the forge app soon.

This should be fine as we can do the same trick with the helper container to copy it into the flowforge.yml file

Reply:
Check here, as you are only setting the key value, not the name, but use the `flowfuse` name later